### PR TITLE
DATAUP-314: minor css fixes

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

The main purpose of this PR was to fix the flex box in the cell control panel so that the job status line was not limited to 450px width. I fixed some other css bits and pieces at the same time.

Commit 1: a couple of global search and replaces:
- in the SCSS files, swapped out `rgb(19x, 19x, 19x)` for `use-color('silver')`
- in the JS files (mainly), searched for `(border.*?) ([a-z]+) solid` and replaced it with `$1 solid $2`

Commit 2 and 3:
- removed some extra DIVs
- changed some class names to be more BEM-some
- made some minor alterations to the app cell layout so it mirrors that of the bulk import cell more closely

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-314
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative and navigating to check out the bulk import and app cells.

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/scss/partials/_bulkImportCell.scss [hunk 1]
```diff
@@ -120,12 +119,6 @@ $button-outdated: rgb(247, 155, 34);
       font-style: italic;
     }
   }
-
```
Comment (ialarmedalien) [inline kbase-extension/scss/partials/_bulkImportCell.scss:123]: this has gone into the `_rcp.scss` file

File: kbase-extension/scss/partials/_buttons.scss [hunk 2]
```diff
@@ -22,35 +22,3 @@
   /* don't add shadows here */
   box-shadow: none;
 }
-
-.kb-btn-action {
```
Comment (ialarmedalien) [inline kbase-extension/scss/partials/_buttons.scss:26]: renamed slightly and moved to `_rcp.scss`

File: kbase-extension/static/kbase/js/widgetApi/narrativeDataWidget.js [hunk 3]
```diff
@@ -16,18 +16,18 @@ define(['bluebird', 'runtimeManager', 'narrativeConfig'], (
...
 ) => {
     'use strict';
-    function factory(config) {
-        var packageName = config.package,
-            packageVersion = config.version,
-            widgetName = config.widget,
-            widgetTitle = config.title,
-            widgetParentNode = config.parent,
-            authRequired = config.authRequired,
+    function factory(conf) {
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/widgetApi/narrativeDataWidget.js:19]: fixed eslint errors in this file

File: nbextensions/appCell2/widgets/appCellWidget.js [hunk 4]
```diff
@@ -772,23 +773,24 @@ define([
 
         // bulkImportCell / cellTabs / buildTabButtons
         function buildRunControlPanelDisplayButtons(events) {
+            const cssBaseClass = 'kb-rcp';
             const buttons = Object.keys(controlBarTabs.tabs)
+                .filter((key) => {
```
Comment (ialarmedalien) [inline nbextensions/appCell2/widgets/appCellWidget.js:778]: Changed this so it mirrors what is in the cellControlPanel widget. At some point this control panel and action stuff can be removed and replaced by the `cellTabs` and `cellControlPanel` widgets; there's just some extra stuff about tab selection that would need to be checked first. Oh, and tests written, of course!

File: nbextensions/appCell2/widgets/appCellWidget.js [hunk 5]
```diff
@@ -893,31 +886,42 @@ define([
...
                                     {
-                                        class: 'kb-app-cell__widget_container',
+                                        class: `${cssCellType}__widget_container`,
                                         dataElement: 'widget',
                                     },
                                     [
                                         div({ class: 'container-fluid' }, [
                                             buildRunControlPanel(events),
                                         ]),
+                                        div(
```
Comment (ialarmedalien) [inline nbextensions/appCell2/widgets/appCellWidget.js:919]: moved the tab-pane + widget to mirror the structure in the bulk import cell. This means changing the path used to access the tab-pane from `run-control-panel.tab-pane` to `body.widget.tab-pane`.

File: nbextensions/appCell2/widgets/appOutputWidget.js [hunk 6]
```diff
@@ -19,17 +19,14 @@ define(['common/runtime', 'common/events', 'common/ui', 'kb_common/html', 'base/
         li = t('li');
 
     function factory(config) {
-        let runtime = Runtime.make(),
+        const runtime = Runtime.make(),
```
Comment (ialarmedalien) [inline nbextensions/appCell2/widgets/appOutputWidget.js:22]: Fixed the eslint errors in this file.

File: test/unit/spec/common/actionButtons-spec.js [hunk 7]
```diff
@@ -6,7 +6,7 @@ define([
     'common/events',
 ], (ActionButton, $, Runtime, UI, Events) => {
     'use strict';
-    let mockActionButton, ui, container;
+    let actionButtonInstance, ui, container;
```
Comment (ialarmedalien) [inline test/unit/spec/common/actionButtons-spec.js:9]: mockActionButton => actionButtonInstance

File: test/unit/spec/common/actionButtons-spec.js [hunk 8]
```diff
@@ -63,21 +64,16 @@ define([
...
         });
 
         it('has a method buildLayout which returns buttons', () => {
-            const mockButtons = mockActionButton.buildLayout(events);
+            const mockButtons = actionButtonInstance.buildLayout(events);
 
-            const $buttonContainer = $(mockButtons).find(
-                '.kb-bulk-import-action-button__container'
-            );
+            const $buttonContainer = $(mockButtons).find('.kb-rcp__action-button-container');
```
Comment (ialarmedalien) [inline test/unit/spec/common/actionButtons-spec.js:74]: fix the class name and DOM structure

**Review by briehl — APPROVED** [review]
Looks like ~90% find/replacing. All good changes. :+1:

**Pull request merged**